pty.c

```diff
@@ -639,16 +639,16 @@ static int ptm_create(int *id)
 	pty->state = MASTER_OPEN | SLAVE_LOCKED;
 	pty->slave_refs = 0;
 
-	object_create(&pty->master, &ptm_ops);
-	object_create(&pty->slave, &pts_ops);
+	posixsrv_object_create(&pty->master, &ptm_ops);
+	posixsrv_object_create(&pty->slave, &pts_ops);
 
-	*id = object_id(&pty->master);
-	oid.port = srv_port();
-	oid.id = object_id(&pty->slave);
+	*id = posixsrv_object_id(&pty->master);
+	oid.port = posixsrv_port();
+	oid.id = posixsrv_object_id(&pty->slave);
 	snprintf(path + sizeof("/dev/pts"), sizeof(PTS_NAME_PADDING), "%d", (int)oid.id);
 
-	object_put(&pty->master);
-/*	object_put(&pty->slave); - Keep one reference for the master */
+	posixsrv_object_put(&pty->master);
+/*	posixsrv_object_put(&pty->slave); - Keep one reference for the master */
```

Comment:
Just a name change. I will stay with the original version (without indentation), I think the author's intention was to highlight this comment so that it would be visible when quickly reviewing the code later.